### PR TITLE
Add comment explaining function creation.

### PR DESCRIPTION
**Pull request by AyuniInaMohamaad**
Add a comment to explain codes that create a function to read the data file.

### CODE REVIEW

File: src/plotting.py [hunk 1]
```diff
@@ -6,6 +6,7 @@
 import matplotlib.pyplot as plt
 import pandas as pd
 
+# Create a function to read the data file
```
Comment (kellybp18) [inline src/plotting.py:9]: Good idea, this is a reasonable change.

Comment (okracht) [inline src/plotting.py:9]: This added comment is beneficial to the script

**Review by kellybp18 — APPROVED** [review]
These are reasonable changes.

**Review by okracht — APPROVED** [review]
Nice clarification of the functions in the script.

**Pull request merged**